src/test/java/api/loans/agetolost/ScheduledAgeToLostApiTest.java

```diff
@@ -73,6 +75,7 @@ public void canAgeTenItemsToLostWhenOverdueByMoreThanInterval() {
       assertThat(itemFromStorage.getJson(), isAgedToLost());
       assertThat(getLoanActions(loanFromStorage), hasAgedToLostAction());
       assertThat(loanFromStorage.getJson(), hasPatronBillingDate(loanFromStorage));
+      assertThat(loansClient.get(overdueLoan).getJson(), hasAgedToLostDate());
```

Comment:
Is this the storage loan or the business logic loan? 

If it is the business logic loan, is this property defined in the relevant JSON schema?

Reply:
Ah, yes. In a fact I don't need to expose the property in business logic loan, but business logic resource just uses storage representation as base and does not remove properties that are not defined in schema.

Should we create a tech debt for it maybe?

Reply:
> Should we create a tech debt for it maybe?

Please do raise a technical debt issue. I believe there already is one for removing the JSON from the domain, which is part of the cause of this issue (and I regret doing it)

Reply:
Here it is https://issues.folio.org/browse/CIRC-915